### PR TITLE
[IMP] l10n_in: add support for SEZ/Export in GSTR-1

### PR DESCRIPTION
**Pull request by hatr-odoo**
This **PR** introduces SEZ/Export support in GSTR-1 for the supply type 'Without Payment of Tax' — where values are included in the form but no tax is applied. This is achieved by creating new taxes exclusively for SEZ/Export.

**task**-3394087

### CODE REVIEW

File: addons/l10n_in/models/template_in.py [hunk 1]
```diff
@@ -87,11 +87,11 @@ def _get_in_account_fiscal_position(self):
             },
         }
 
-    def _get_l10n_in_fiscal_tax_vals(self, use_zero_rated_igst=False):
+    def _get_l10n_in_fiscal_tax_vals(self, use_zero_rated_igst=False, use_sez_exp_igst=False):
         return [Command.clear()] + [
             Command.create({
                 'tax_src_id': f"sgst_{tax_type}_{rate}",
-                'tax_dest_id': f"igst_{tax_type}_{rate if not use_zero_rated_igst else 0}",
+                'tax_dest_id': f"igst_{tax_type}_{rate if not use_zero_rated_igst else 0}{'_sez_exp' if use_sez_exp_igst and not use_zero_rated_igst else ''}",
```
Comment (hamo-odoo) [inline addons/l10n_in/models/template_in.py:94]:
```suggestion
                'tax_dest_id': f"igst_{tax_type}_{rate if not use_zero_rated_igst else 0}{'_sez_exp' if use_sez_exp_igst else ''}",
```

Comment (hatr-odoo) [reply addons/l10n_in/models/template_in.py]: Hello @hamo-odoo, I have made the requested changes.

**Review by hamo-odoo — APPROVED** [review]
@jva-odoo @jco-odoo LGTM

File: addons/l10n_in/data/template/account.account-in.csv [hunk 2]
```diff
@@ -5,8 +5,9 @@
 "p10051","SGST Receivable","10051","asset_current","l10n_in.sgst_tag_account","False"
 "p10052","CGST Receivable","10052","asset_current","l10n_in.cgst_tag_account","False"
 "p10053","IGST Receivable","10053","asset_current","l10n_in.igst_tag_account","False"
+"p10054","IGST SEZ/Export Control Account","10054","asset_current","","False"
 "p10057","Reverse Charge Tax Receivable","10057","asset_current","","False"
-"p10054","TDS Receivable","10058","asset_current","","False"
+"p10058","TDS Receivable","10058","asset_current","","False"
```
[the remark this answers is not in the document]
Comment (jva-odoo) [reply addons/l10n_in/data/template/account.account-in.csv:10]: Wrong XML id so we need to fix it.
so my suggestion is to write a migration script so the name is not changed
in-migration we just renamed existing p10054 to p10058

**Comment by ajjo-odoo** [comment]
thanks for your response,
i have tried but couldn’t find any other way.

Regards,
Ajaykumar Jobanputra
Product Owner - Account


Odoo India, 401 & 402, 4th floor, Tower 3, Infocity

Gandhinagar ( Gujarat ) Pin - 382007

**Comment by jco-odoo** [comment]
@hatr-odoo So waiting for the upgrade script here

**Comment by hamo-odoo** [comment]
@jco-odoo I have rebased the branch and I have created the upgrade PR which is ready for review

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**